R/data_type.R

```diff
@@ -44,25 +44,30 @@ identify_data_type <- function(df,
                                codilution_conc = 2,
                                matrix_conc = 1) {
   
-  # find the pairs of drugs with relevant metadata
```

Comment:
Maybe that would be a god idea to rename `df` in something more descriptive? `dt_response`?

Reply:
renamed into dt

Reply:
✔️